build.gradle

```
@@ -66,8 +66,13 @@ allprojects {
 
         tasks.withType(Test).configureEach {
             useJUnitPlatform(){
-                excludeTags project.hasProperty('runUnstableTests') ? 'runAllTestsNoExclusions' : 'Unstable'
-                excludeTags project.hasProperty('runMultiClusterTests') ? 'runAllTestsNoExclusions' : 'MultiCluster'
+                if (project.hasProperty('runMultiClusterTests')) {
+                    includeTags 'MultiCluster'
+                } else if (project.hasProperty('runUnstableTests')) {
+                    includeTags 'runUnstableTests'
+                } else {
+                    excludeTags 'MultiCluster', 'runUnstableTests'
```

```suggestion
                    excludeTags 'MultiCluster', 'Unstable'
```

we need to refrecn the unit tag here, which is `Unstable` , runUnstableTests is the gradle property which controls this

---

I've applied this